### ISSUE TITLE
Big Files

### ISSUE THREAD

**Issue by hurzl** [tudelft3d/CityGML2OBJs#6]
My GML File is 1 GB and I killed python when it reached 4GB of RAM.
Grouping or not.

**Comment by fbiljecki** [comment]
Yes, it can take resources. Unfortunately I haven't found a way how to make it more efficient.

**Comment by hurzl** [comment]
Couldn't it handle each building separately?

**Comment by fbiljecki** [comment]
But then you would end up in one OBJ file for each building. Maybe that could be an alternative solution, but at the moment I have paused the development due to other work.
